.github/workflows/ci.yml

```
@@ -119,6 +119,34 @@ jobs:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
       - name: CI snapshot tests for c4p
         run: yarn ci-check:test:c4p:snapshot
+  test-c4p-react-major-version:
+    strategy:
+      matrix:
+        react: [17]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20.x'
+          cache: yarn
+      - name: Cache dependencies
+        uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 #4.2.0
+        with:
+          path: '**/node_modules'
+          key:
+            test-c4p-react-major-versions-modules-${{ runner.os }}-${{
+            hashFiles('yarn.lock') }}
+      - name: Install
+      - run: yarn
```

```suggestion
        run: yarn
```

I saw when the action ran in your fork there was some yml issues, `run` shouldn't need the dash since it's supposed to be within `- name: Install`